package-lock.json

```diff
@@ -3192,7 +3192,7 @@
     },
     "ansi-escapes": {
       "version": "3.1.0",
-      "resolved": "https://registry.npmjs.org/ansi-escapes/-/ansi-escapes-3.1.0.tgz",
+      "resolved": "http://registry.npmjs.org/ansi-escapes/-/ansi-escapes-3.1.0.tgz",
```

Comment:
I think the `package-lock.json` changes can be omitted and will make this PR a bit smaller.